tests/foreman/cli/test_contentview.py

```diff
@@ -4793,6 +4793,28 @@ def test_positive_incremental_update_propagate_composite(self):
             puppet_module['id']
         )
 
+    @tier3
+    def test_positive_katello_reimport(self):
+        """ Close loop bug for running katello:reimport.  Making sure
+        that katello:reimport works and doesn't throw an error.
+
+        :id: b4119265-1bf0-4b0b-8b96-43f68af39708
+
+        :Steps: Have satellite up and run 'foreman-rake katello:reimport'
+
+        :expectedresults: Successfully reimport without errors
+
+        :CaseAutomation: automated
+
+        :CaseImportance: Medium
+
+        :bz: 1771555
+        """
+
+        result = ssh.command('foreman-rake katello:reimport')
```

Comment:
I have some questions here is that It is necessary to add some test case where we are testing command which is not used by the Customer in the first place?  As per my understanding, this is used by developers to resolve the issues or debug the issues and it is a low-level utility tool.  Do we really want to automate such cases?

If Yes, I would say put it into the destructive test.

Reply:
This bugzilla has customer case and knowledge base article linked; it's clone of another bugzilla, that has two more customer cases linked. Clearly **some** customers use that.

My guess is it's part of some knowledge base article around debugging or presented as solution to some specific problem.

Reply:
If it is used by the customer and mentioned in the documentation then I don't have any argument. I just wanted to be clear that we are supporting this command as a usual case.